1-JavaScript/Week3/homework/ex3-lemonAllergy.test.js

```diff
@@ -26,26 +26,35 @@ const fruitBasket = [
 ];
 
 // ! Function under test
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function sanitizeFruitBasket(basket, allergicFruit) {
+  const sanatizedBasket = basket.filter((fruit) => {
+    if (fruit !== allergicFruit) {
+      return fruit;
+    } else {
+      console.log('You are dead');
+    }
+  });
+  return sanatizedBasket;
```

Comment:
Funny implementation of this exercise 😆 It works of course but I would recommend to refactor this code a little bit. You can return it all in one line and use an "implicit return" in the arrow function 🙂 

```suggestion
  return basket.filter((fruit) => fruit !== allergicFruit
```